### ISSUE TITLE
Unable to interact with titlebar or window chrome of editor

### ISSUE THREAD

**Issue by redxdev** [godotengine/godot#15833]
**Godot version:**
ff59c56 mono x64

**OS/device including version:**
Windows 10 x64

**Issue description:**
The titlebar and window chrome (including the close/minimize/maximize buttons and the edges of the window used to resize the window) are sometimes impossible to interact with. When this situation happens, the window cannot be moved via the titlebar, the titlebar buttons do not show hover effects and cannot be clicked, and the window cannot be resized.

Whenever Godot is exhibiting this behavior, opening up a different window and then going back to the editor seems to fix it temporarily. On the other hand, clicking anywhere in the editor causes this to happen once again.

**Steps to reproduce:**
The issue is intermittent, and I'm not actually sure what causes this. Once it starts happening, I can reproduce the effect 100% of the time (see above) as long as the editor isn't closed. Upon a fresh start of the editor, the issue disappears and I'm not sure what causes it to happen again.

**Comment by akien-mga** [comment]
Can you post a screenshot of the issue? Also, what's your GPU and drivers version? There are known bugs with older Intel drivers on Windows 10, if that's your situation try to upgrade the drivers.

**Comment by Zylann** [comment]
I had this problem too sometimes, I don't know how it happens.

**Comment by Calinou** [comment]
I could also reproduce this on Windows (with a NVIDIA graphics card), but not on Linux.

**Comment by redxdev** [comment]
I am using an Nvidia GTX 1080. I'll check the driver version when I'm back at my computer, and if I can reproduce it I'll take a video.

**Comment by vnen** [comment]
Yeah, I've seen this too. I'm using an AMD RX 480, so it's likely not a drivers fault. I had a similar issue with other applications so I thought it's not related to Godot, but maybe it is.

**Comment by vnen** [comment]
BTW, I can temporarily solving it by unmaximizing the window (using the taskbar context menu) and then maximizing again.

**Comment by redxdev** [comment]
I've had that temporarily solve it as well, only to start happening again the moment I click somewhere within the editor.

**Comment by damarindra** [comment]
I thought someone has reported this issue since I've been there with godot 2. The solution is alt+tab and select the title bar.

**Comment by binbitten** [comment]
In os_windows, WM_MOUSEHWHEEL is not properly handled which will make the capture counter wrong and screw the title bar. Some mice have horizontal tilt on the scroll wheel which can be really easy to unknowingly trigger. Has anyone experienced this on any other platform than windows and without a horiz. tilt on the scroll wheel?

**Comment by redxdev** [comment]
Well, I can confirm that using the the horizontal scroll on my mouse is what causes this (on windows). Now I can trigger this bug in the editor _and_ in games (which makes sense).

**Comment by Zylann** [comment]
This still happens in 3.0.6 for me:
https://youtu.be/wL2k_mHxkxI

**Comment by justinbowes** [comment]
I am experiencing this, and it just started happening after I tried to use horizontal scroll. v3.2.2.stable.official.

**Comment by Calinou** [comment]
@binbitten Are you still around?

**Comment by binbitten** [comment]
I'm positively certain that this piece of code is a or _the_ culprit: 
[https://github.com/godotengine/godot/blob/80a8a36ca19f9d2bc9fad83f671d732ea339a93f/platform/windows/display_server_windows.cpp#L2461-L2471](https://github.com/godotengine/godot/blob/80a8a36ca19f9d2bc9fad83f671d732ea339a93f/platform/windows/display_server_windows.cpp#L2461-L2471)

The _pressrc_-counter will cause trouble as it depends on events always being correctly fired. If and when it gets out of "sync" the mouse capture won't be released. I'm not quite sure of the snippet's intended functionality; it seems rather pointless, but it sure needs to be implemented another way. This will also fix #33928.

**Comment by KoBeWi** [comment]
Is this still happening in Godot 4?
